graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlJcrProperty.java

```diff
@@ -313,6 +313,23 @@ public Long getLongValue() {
         }
     }
 
+    /**
+     * @return The binary size of the JCR node as a Long, null otherwise
+     */
+    @GraphQLField
+    @GraphQLName("size")
+    @GraphQLDescription("The binary size of the JCR node as a Long, null otherwise")
+    public Long getSize() {
+        try {
+            if (property.isMultiple()) {
```

Comment:
You should test the type of the property here as well and return null if not a BINARY.